src/Arcus.Scripting.ApiManagement/Scripts/Upload-AzApiManagementCertificate.ps1

```diff
@@ -0,0 +1,13 @@
+param(
+    [Parameter(Mandatory = $true)][string] $ResourceGroupName,
+    [Parameter(Mandatory = $true)][string] $ServiceName,
+    [Parameter(Mandatory = $true)][string] $CertificateFilePath,
+    [Parameter(Mandatory = $true)][string] $CertificatePassword
```

Comment:
Could you include throwing an error message in case 1 of the properties/values has not been provided?

Reply:
Done that, thanks for reminding. Resolving.